trackintel/io/file.py

```diff
@@ -472,6 +472,10 @@ def read_trips_csv(*args, columns=None, tz=None, index_col=object(), **kwargs):
         if not pd.api.types.is_datetime64tz_dtype(trips[col]):
             trips[col] = _localize_timestamp(dt_series=trips[col], pytz_tzinfo=tz, col_name=col)
 
+    # convert to geodataframe
+    trips["geom"] = trips["geom"].apply(wkt.loads)
```

Comment:
Here one would have to add an if clause to keep the compatibility with dataframes (compared to geodataframes)